stable/spark/values.yaml

```diff
@@ -90,3 +99,20 @@ Zeppelin:
       StorageClass: "-"
       Size: 10G
       AccessMode: ReadWriteOnce
+
+Livy:
```

Comment:
What does Livy enable and why do I need it?

Reply:
Livy provides a rest API on top of spark submit. It’s very useful for submitting jobs to spark without needing to install anything locally. We need this for many of our ongoing projects.